slides/src/content/unit-testing.md

```diff
@@ -89,17 +100,17 @@ describe('CapitalizePipe', () => {
 ```
 
 ---
-<!-- .slide: id="unit-testing-instantiate-fixture" -->
-## Instantiate the Fixture in `beforeEach`
+<!-- .slide: id="unit-testing-instantiate-pipe" -->
+## Instantiate the Pipe in `beforeEach`
 
-- We will need a fixture for each test, so don't duplicate that code
+- We will need an instance for each test, so instantiate it in the `beforeEach` to avoid duplicating code
 
 #####_src/app/capitalize.pipe.spec.ts_
 ```ts
 describe('CapitalizePipe', () => {
 
   let pipe;
-
+  
```

Comment:
please strip trailing whitespace off lines (it sometimes makes Markdown think we want a `<br>`